datakyt/db/create_table.sql

```diff
@@ -0,0 +1,152 @@
+SET ROLE datakyt_admin;
+
+-- Table description
+CREATE TABLE project
+(
+	id	        SMALLSERIAL 	PRIMARY KEY,
+	name		char(65)	    NOT NULL
+
+);
+
+
+CREATE TABLE office
+(
+	id	        SMALLSERIAL 	PRIMARY KEY,
+	city		char(255)	    NOT NULL
+
+);
+
+
+CREATE TABLE employee
+(
+	id		SMALLSERIAL 	PRIMARY KEY,
+	name		char(65)	NOT NULL,
```

Comment:
We have special data type `name`, Please check if we can use it everywhere 
https://www.postgresql.org/docs/current/datatype-character.html#DATATYPE-CHARACTER-SPECIAL-TABLE
CC @paulzcooper

Reply:
I have read some sources and they recommend using a text type, so I changed all lines of char to text in the name.